src/rust/engine/fs/store/src/lib.rs

```diff
@@ -613,6 +613,19 @@ impl Store {
             Some((path, file_node.digest.as_ref().unwrap().try_into().unwrap())),
           )
         }));
+        path_stats.extend(directory.symlinks.iter().map(move |link_node| {
```

Comment:
I doubt we're adding more node types, but this is why I prefer `match`, so when we add new types it's obvious what needs to be updated

Reply:
Agreed. But this is a protobuf, and it's not defined in terms of an enum.